app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CommentsController < ApplicationController
+  before_action :set_commentable
+  before_action :set_comment, only: [:destroy, :edit, :update]
+
+  def index
+    @comments = @commentable.comments
+  end
+
+  def new
+    @comments = @commentable.comments.recent.limit(5)
+    @comment = @commentable.comments.new
+  end
+
+  def create
+    @comment = @commentable.comments.new(comment_params.merge(user_id: current_user.id))
```

Comment:
i guess its better if we take this to new line.
@comment.user_id = current_user.id